src/_pytest/fixtures.py

```diff
@@ -1415,10 +1435,14 @@ class FixtureManager:
     def __init__(self, session: "Session") -> None:
         self.session = session
         self.config: Config = session.config
-        self._arg2fixturedefs: Dict[str, List[FixtureDef[Any]]] = {}
-        self._holderobjseen: Set[object] = set()
+        # Maps a fixture name (argname) to all of the FixtureDefs in the test
+        # suite defined with this name. Populated by parsefactories().
```

Comment:
Including external plugins, correct?

Reply:
Correct

Reply:
Thanks, might be worth clarifying that then:

```suggestion
        # suite and external plugins defined with this name. Populated by parsefactories().
```